src/cmdlib.sh

```diff
@@ -299,11 +299,21 @@ openshift_git_hack() {
 
     # Some versions of Openshift do not include the GIT repo in the checkout.
     # This may be needed for Openshift versions 3.9 and earlier.
-    # See https://github.com/coreos/coreos-assembler/issues/320.
-
-    # $OPENSHIFT_GIT_HACK, $GIT_REF and $GIT_URL are environment variables,
-    # provided by Openshift or the container run time. See
-    # https://github.com/coreos/coreos-assembler/pull/324 for an example.
+    # See https://github.com/coreos/coreos-assembler/issues/320 and
```

Comment:
@dustymabe okay, I put more documentation in regarding the context. How does this look?

Reply:
LGTM